src/theories/Limits.jl

```diff
@@ -56,32 +56,33 @@ ideas". Strictly speaking, this theory is not of a "finitely complete category"
 (a category in which finite limits exist) but of a "category with *chosen*
 finite limits".
 """
-@theory ThCompleteCategory{Ob,Hom,Terminal,Product,Equalizer} <:
-    ThCategoryWithProducts{Ob,Hom,Terminal,Product} begin
-  Equalizer(f::(A → B), g::(A → B))::TYPE ⊣ (A::Ob, B::Ob)
+@theory ThCompleteCategory <: ThCategoryWithProducts begin
+  Equalizer(f::(A → B), g::(A → B))::TYPE ⊣ [A::Ob, B::Ob]
   
   # Equalizers.
-  equalizer(f::(A → B), g::(A → B))::Equalizer(f,g) ⊣ (A::Ob, B::Ob)
-  ob(eq::Equalizer(f,g))::Ob ⊣ (A::Ob, B::Ob, f::(A → B), g::(A → B))
+  equalizer(f::(A → B), g::(A → B))::Equalizer(f,g) ⊣ [A::Ob, B::Ob]
+  ob(eq::Equalizer(f,g))::Ob ⊣ [A::Ob, B::Ob, f::(A → B), g::(A → B)]
   (incl(eq::Equalizer(f,g))::(ob(eq) → A)
-    ⊣ (A::Ob, B::Ob, f::(A → B), g::(A → B)))
+    ⊣ [A::Ob, B::Ob, f::(A → B), g::(A → B)])
   (factorize(eq::Equalizer(f,g), h::(C → A),
              eq_h::Equalizer(h⋅f,h⋅g))::(ob(eq_h) → ob(eq))
-    ⊣ (A::Ob, B::Ob, C::Ob, f::(A → B), g::(A → B)))
+    ⊣ [A::Ob, B::Ob, C::Ob, f::(A → B), g::(A → B)])
   
   # Equalizer axioms.
-  (incl(eq)⋅f == incl(eq)⋅g
-    ⊣ (A::Ob, B::Ob, f::(A → B), g::(A → B), eq::Equalizer(f,g)))
-  (incl(eq) == id(A)
-    ⊣ (A::Ob, B::Ob, f::(A → B), eq::Equalizer(f,f)))
-  (factorize(eq,h,eq_h) ⋅ incl(eq) == incl(eq_h) ⋅ h
-    ⊣ (A::Ob, B::Ob, C::Ob, f::(A → B), g::(A → B), h::(C → A),
-       eq::Equalizer(f,g), eq_h::Equalizer(h⋅f, h⋅g)))
-  (factorize(eq, k⋅incl(eq), eq_k) == k
-    ⊣ (A::Ob, B::Ob, D::Ob, f::(A → B), g::(A → B), eq::Equalizer(f,g),
-       k::(D → ob(eq)), eq_k::Equalizer(k⋅incl(eq)⋅f, k⋅incl(eq)⋅g)))
+  # (incl(eq)⋅f == incl(eq)⋅g
```

Comment:
What happened to these axioms? I don't immediately see why they are different from others, hence would have broken.

Reply:
Uncommenting them doesn't break anything, so I uncommented them.